docs/install.rst

```diff
@@ -233,6 +236,23 @@ Nix provides atomic upgrades and rollbacks, it's reliable and reproducible thank
 
 .. _pragmatic_installation:
 
+☤ Mac Installation of Pipenv
+===============================
+
+Homebrew is a popular open-source package management system for macOS.
+
+Installing pipenv via homebrew will keep pipenv and all of its dependencies in
```

Comment:
Homebrew is capitalized in the official documentation and we should be making sure we use that consistently. There are multiple places in this PR this needs to be tweaked.